azure/firewall-policy/.tflint.hcl

```diff
@@ -0,0 +1,4 @@
+plugin "terraform" {
```

Comment:
Might be best to add azurerm linter here too

Reply:
added azurerm plugin and ran